src/utils.ts

```diff
@@ -647,6 +648,32 @@ function isValidSnapshot(snapshot: number | string, network: string) {
   );
 }
 
+export function isStarknetAddress(address: string): boolean {
+  if (!address) return false;
+
+  try {
+    validateAndParseAddress(address);
+    return true;
+  } catch (e: any) {
+    return false;
+  }
+}
+
+export function isEvmAddress(address: string): boolean {
+  return isAddress(address);
+}
+
+export function getFormattedAddress(
+  address: string,
+  format: string[] = ['evm', 'starknet']
+): string {
+  if (format.includes('evm') && isAddress(address)) return getAddress(address);
+  if (format.includes('starknet') && isStarknetAddress(address))
+    return validateAndParseAddress(address);
+
+  throw new Error(`Invalid address: ${address}`);
+}
```

Comment:
`format` can either be evm or starknet right? is there a reason to use array? an address can't be both no? 

Also on sequencer we are no passing any `format` param here 🤔

`snapshot.utils.getFormattedAddress(legacyBody.address);`

Reply:
Correct, fixed also on sequencer